bin/app.js

```diff
@@ -59,6 +60,8 @@ function App(config) {
     maxAge: config.statics.maxAge
   }))
 
+  app.use('/style.json', new mapStyle(config, languages))
```

Comment:
I think we should use express compression on this endpoint.  
The gzipped style is 8KB, compared to 100KB without compression

Reply:
sure, i'll look for the how to

Reply:
compression (the most used middleare) enable gzip for all request, it take a filter param but i think it's dirty way. 
or i can enable compression for all request

Reply:
didn't know about it 
the force is fronce in this one